foreman.te

```diff
@@ -466,3 +466,17 @@ typealias bin_t alias foreman_hook_t;
 # the plugin daemon uses daemon gem for the backround job
 type foreman_tasks_exec_t;
 init_daemon_domain(passenger_t, foreman_tasks_exec_t)
+
+######################################
+#
+# Foreman Memcache plugin
+#
+
+optional_policy(`
+    tunable_policy(`passenger_run_foreman', `
+        require{
+            type memcache_port_t;
+        }
+        allow passenger_t memcache_port_t:tcp_socket name_connect;
```

Comment:
Thanks. Please change the four lines above to:

    corenet_tcp_connect_memcache_port(passenger_t)